src/components/map/MapService.js

```diff
@@ -375,7 +376,7 @@ goog.require('ga_urlutils_service');
           var wmsOptions = {
             url: getCapLayer.wmsUrl,
             label: getCapLayer.Title,
-            extent: getCapLayer.extent,
+            extent: gaMapUtils.mergeWithDefaultExtent(getCapLayer.extent),
```

Comment:
Hmm I'm not sure of this one, if the layer's extent define by the GetCap is 10km aroud Zurich for example. I don't think we want the layer was requested outside this extent